src/apps/front-office/design-system/hooks/index.ts

```diff
@@ -0,0 +1,23 @@
+/**********************************************************/
+/**************** Custom Hook *****************************/
+/**********************************************************/
+
+import { useEffect, useState } from "react";
+
+export function useWindowScroll() {
```

Comment:
Nice hook, I'd recommend writing a doc-block over the hook to illustrate the purpose of the function instead of the `Custom Hook` comment above.